eng/testing/tests.browser.targets

```diff
@@ -112,6 +112,7 @@
     <_XHarnessArgs Condition="'$(WasmXHarnessTestsTimeout)' != ''" >$(_XHarnessArgs) &quot;--timeout=$(WasmXHarnessTestsTimeout)&quot;</_XHarnessArgs>
     <_XHarnessArgs Condition="'$(WasmXHarnessArgsCli)' != ''"      >$(_XHarnessArgs) $(WasmXHarnessArgsCli)</_XHarnessArgs>
 
+    <_AppArgs Condition="'$(MonoWasmBuildVariant)' == 'multithread'">$(_AppArgs) -threads</_AppArgs>
```

Comment:
Consider Ilona's https://github.com/dotnet/runtime/pull/97560

Reply:
Lets see which goes in first